plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -178,6 +180,10 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni DefaultFun where
         makeBuiltinMeaning
             ((<=) @Integer)
             (runCostingFunTwoArguments . paramLessThanEqualsInteger)
+    toBuiltinMeaning ShiftInteger =
+        makeBuiltinMeaning
+            (\x i -> Bits.shift @Integer x (fromIntegral @Integer i))
```

Comment:
I guess `fromIntegral` is incorrect here just like in case with `ConsByteString` and should be checked manually?

Reply:
`Integer` has a `Bits` instance, so why do we need to do this?